airbyte-webapp/src/views/Connection/CatalogTree/FieldRow.tsx

```diff
@@ -29,13 +30,20 @@ const LastCell = styled(Cell)`
   margin-right: -10px;
 `;
 
+function humanize(str: string) {
+  return str
+    .replace(/^[\s_]+|[\s_]+$/g, " ")
+    .replace(/[_\s]+/g, " ")
+    .replace(/^[a-z]/, (m) => m.toUpperCase());
+}
+
 const FieldRowInner: React.FC<FieldRowProps> = ({ onPrimaryKeyChange, onCursorChange, path, ...props }) => {
   return (
     <>
       <FirstCell ellipsis flex={1.5}>
         <NameContainer title={props.name}>{props.name}</NameContainer>
       </FirstCell>
-      <DataTypeCell nullable={props.nullable}>{props.type}</DataTypeCell>
+      <DataTypeCell nullable={props.nullable}>{humanize(props.format ?? props.type)}</DataTypeCell>
```

Comment:
I think it would be better if we actually map this to an internationalized string. Looking at the [supported datatypes](https://docs.airbyte.com/understanding-airbyte/supported-data-types#the-types) we have some that are not as easily "humanizable" from the format and type. I'd suggest we add a utility method that basically turns the JSON schema of that property into an `i18n`ed string, so that we'd also be able to translate types like "Datetime without timezone".

Reply:
Also that utility could be perfectly unit tested :)